src/dbg/_exports.cpp

```diff
@@ -169,7 +169,7 @@ static bool getLabel(duint addr, char* label)
             duint start;
             if(FunctionGet(addr, &start, nullptr) && addr == start)
             {
-                sprintf_s(label, MAX_LABEL_SIZE, "sub_%" fext "X", start);

```

Comment:
Actually this changes the behavior of how the string is printed. The idea was unpadded and now it will be padded.